test/end-to-end/pages/applicant/phone.js

```diff
@@ -2,18 +2,14 @@
 
 const commonContentEn = require('app/resources/en/translation/common');
 const commonContentCy = require('app/resources/cy/translation/common');
-const phoneContentEn = require('app/resources/en/translation/applicant/phone');
-const phoneContentCy = require('app/resources/cy/translation/applicant/phone');
 
 module.exports = async function(language = 'en') {
     const I = this;
     const commonContent = language === 'en' ? commonContentEn : commonContentCy;
-    const phoneContent = language === 'en' ? phoneContentEn : phoneContentCy;
 
     await I.checkPageUrl('app/steps/ui/applicant/phone');
-    if (language === 'en') {
-        await I.waitForText(phoneContent.phoneNumber);
-    }
-    await I.fillField('#phoneNumber', '123456789');
+    const locator = {css: '#phoneNumber'};
```

Comment:
We've taken out a check for  English phone number label - shouldn't it check both English and Welsh phone number label

Reply:
done.